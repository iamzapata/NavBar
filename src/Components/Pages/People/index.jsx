import React from "react"
import PageContainer from "Pages/PageContainer"

const People = () => <PageContainer>Hello People</PageContainer>

export default People
