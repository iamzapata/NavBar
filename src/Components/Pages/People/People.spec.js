import React from "react"
import { render, cleanup } from "react-testing-library"
import People from "./index"

describe("<People />", () => {
  afterEach(cleanup)
  it("Should render self", () => {
    const { container } = render(<People />)
    expect(container.firstChild).toBeTruthy()
    expect(container.firstChild).toMatchSnapshot()
  })
})
