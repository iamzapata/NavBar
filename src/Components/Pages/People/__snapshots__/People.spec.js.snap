// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<People /> Should render self 1`] = `
<div
  class="PageContainer"
>
  Hello People
</div>
`;
